src/utils/solanaUtils.ts

```diff
@@ -1626,3 +1626,18 @@ export const submitSolana = async ({
 
   return txid
 }
+
+export const parseTransactionError = (
+  balance: Balance<AnyCurrencyType>,
+  message?: string,
+) => {
+  if (!message) return 'UNKNOWN ERROR'
+
+  // Check if error has instructionError from {"InstructionError":[1,{"Custom":1}]}
+  const instructionError = message.includes('InstructionError')
+  if (instructionError && balance.floatBalance > 0.02) {
+    return 'The SOL balance on this account is too low to complete this transaction'
```

Comment:
Won't be translated, but maybe ok for now.

Reply:
Yeah let’s leave as is for now